ci/scripts/driver_weekly.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+set -eux
+
+##############################################################################################
+#
+# Script description: Top level driver script for running
+#                     weekly CI regression tests
+#
+# Abstract:
+#
+# This script runs the high resolution cases found in $HOMEgfs/ci/cases/weekly
+# from the develop branch for the global-workflow repo that are intended to run on a weekly basis
+# from a cron job. When run it will clone and build a new branch from the EMC's global-workflow and
+# and create a pr using GitHub CLI by moving and replacing the yaml case files in
+# ${HOMEgfs}/ci/cases/weekly to {HOMEgfs}/ci/cases/pr.  Then at point it can simply also add the
+# the requisite labels so that current BASH CI framework can run these cases.
+##############################################################################################
+
+#################################################################
+# TODO using static build for GitHub CLI until fixed in HPC-Stack
+#################################################################
+export GH=${HOME}/bin/gh
+export REPO_URL=${REPO_URL:-"https://github.com/NOAA-EMC/global-workflow.git"}
+
+################################################################
+# Setup the relative paths to scripts and PS4 for better logging
+################################################################
+HOMEgfs="$(cd "$(dirname  "${BASH_SOURCE[0]}")/../.." >/dev/null 2>&1 && pwd )"
+scriptname=$(basename "${BASH_SOURCE[0]}")
+echo "Begin ${scriptname} at $(date -u)" || true
+export PS4='+ $(basename ${BASH_SOURCE[0]})[${LINENO}]'
+
+#########################################################################
+# Set up runtime environment variables for accounts on supported machines
+#########################################################################
+
+source "${HOMEgfs}/ush/detect_machine.sh"
+case ${MACHINE_ID} in
+  hera | orion)
+    echo "Running Automated Testing on ${MACHINE_ID}"
+    source "${HOMEgfs}/ci/platforms/${MACHINE_ID}.sh"
+    ;;
+  *)
+    echo "Unsupported platform. Exiting with error."
+    exit 1
+    ;;
+esac
```

Comment:
Can you explain why this block is required?

Reply:
We still need a common GFS_CI_ROOT accross all platforms so we can run it that one time anywhere

Reply:
Ok.  I see `GFS_CI_ROOT` is used in this script to define the location of `develop_temp`.  That location could as well be in `/tmp`, but ok!